src/test/java/uk/gov/companieshouse/registeredemailaddressapi/integration/RegisteredEmailAddressControllerIntegrationTest.java

```diff
@@ -49,6 +50,7 @@ class RegisteredEmailAddressControllerIntegrationTest {
     protected TransactionService transactionService;
 
     @MockBean
+    @Qualifier("oracleQueryApiDataRetrievalServiceImpl")
     protected PrivateDataRetrievalService privateDataRetrievalService;
```

Comment:
(why protected? etc other comments as per new test, but not your problem for this existing test)